### PR TITLE
Completed test coverage for RedirectFallbackMiddleware.

### PR DESCRIPTION
**Pull request by desecho**

### CODE REVIEW

**Comment by sir-sigurd** [comment]
https://docs.djangoproject.com/en/dev/internals/contributing/committing-code/#committing-guidelines
> Write detailed commit messages in the **past** tense, not present tense.

**Comment by desecho** [comment]
Fixed. Thanks.

File: tests/redirects_tests/tests.py [hunk 1]
```diff
@@ -35,6 +35,17 @@ def test_redirect_with_append_slash_and_query_string(self):
...
         self.assertRedirects(response, '/new_target/', status_code=301, target_status_code=404)
 
+    def test_skip_redirect_check(self):
+        """Skip redirect check for non-404 requests."""
+        response = self.client.get('/')
+        self.assertEqual(response.status_code, 200)
+
+    @override_settings(APPEND_SLASH=True)
+    def test_redirect_not_found(self):
+        """Skip redirect check for non-404 requests."""
```
Comment (timgraham) [inline tests/redirects_tests/tests.py]: Typo that this is the same as previous?

File: tests/redirects_tests/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from .views import normal_view
+
+urlpatterns = [
+    url(r'^$', normal_view),
```
Comment (timgraham) [inline tests/redirects_tests/urls.py]: I think you could skip the views file and use `lambda req: HttpResponse('OK')` instead.

File: tests/middleware_exceptions/urls.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 from . import views
 
+
```
Comment (timgraham) [inline tests/middleware_exceptions/urls.py]: revert

**Pull request merged**